src/textual/app.py

```diff
@@ -160,6 +164,11 @@ def __init__(
         if css is not None:
             self.css = css
 
+        _parse_features = os.getenv("TEXTUAL", "") if _features is None else _features
+        self.features = set(
```

Comment:
Maybe we could type `self.features` so that by looking at this code we would instantly see what feature flags are supported by Textual, rather than having to grep `in self.features` on the source code?
i.e. something like:
```py
FeatureFlag = Literal['debug', 'devtools'] # at the module level

...
...
        self.features: set[FeatureFlag] = set(
           ...
        )

Reply:
I think that's probably a good idea.

Reply:
Ah, don't mind me! `typing.Literal` was added in Python 3.8, and we're targetting 3.7+ :pensive: 
We could use an Enum instead, but that would make the code less "smooth" than using literals imho?

Reply:
DOne!

Reply:
You can import it from `typing_extensions`. Makes for ugly imports but it can work in 3.7